src/tests/ftest/util/pool_security_test_base.py

```diff
@@ -28,6 +28,7 @@
 from daos_utils import DaosCommand
 from dmg_utils import DmgCommand
 import dmg_utils
+import agent_utils as agu
```

Comment:
```suggestion
from agent_utils import include_local_host
```
It's better to import just the method.